transforms/code/malware/install_clamd_mac.sh

```diff
@@ -1,26 +1,35 @@
+#!/bin/bash
+
+# Function to copy and edit config files
+copy_and_edit_config() {
+    local source="$1"
+    local destination="$2"
+
+    # Copy the config file
+    cp -f "$(brew --prefix)/etc/clamav/$source.sample" "$destination"
+
+    # Edit the config file
+    sed -i '' -e 's/^Example/# Example/' "$destination"
+    echo "DatabaseDirectory /var/lib/clamav" >> "$destination"
+}
+
 # Install ClamAV with Homebrew
 brew install clamav
 
-# Copy and edit config files.
-\cp -f $(brew --prefix)/etc/clamav/clamd.conf.sample $(brew --prefix)/etc/clamav/clamd.conf
-sed -i '' -e 's/^Example/# Example/' $(brew --prefix)/etc/clamav/clamd.conf
-echo "DatabaseDirectory /var/lib/clamav" >> $(brew --prefix)/etc/clamav/clamd.conf
-echo "LocalSocket /var/run/clamav/clamd.ctl" >> $(brew --prefix)/etc/clamav/clamd.conf
-\cp -f $(brew --prefix)/etc/clamav/freshclam.conf.sample $(brew --prefix)/etc/clamav/freshclam.conf
-sed -i '' -e 's/^Example/# Example/' $(brew --prefix)/etc/clamav/freshclam.conf
-echo "DatabaseDirectory /var/lib/clamav" >> $(brew --prefix)/etc/clamav/freshclam.conf
-
-# Create a directory for a local unix socket
-if [ ! -e /var/run/clamav ]; then
-    sudo mkdir -p /var/run/clamav
-fi
-sudo chown $(id -u):$(id -g) /var/run/clamav
-
-# Create a direcotry for a database of ClamAV
-if [ ! -e /var/lib/clamav ]; then
-    sudo mkdir -p /var/lib/clamav
-fi
-sudo chown $(id -u):$(id -g) /var/lib/clamav
-
-# Update a database of ClamAV
+# Copy and edit clamd.conf
```

Comment:
This all looks very good.  However, when I run this on a Mac M1, it fails `make test-src`.  If i use the original version it does not fail.  I tried adding back the `\cp` in place of `cp`, but no help.
```
...
Warning: clamav 1.3.1 is already installed and up-to-date.
To reinstall 1.3.1, run:
  brew reinstall clamav
ClamAV update process started at Fri May  3 09:24:24 2024
daily.cld database is up-to-date (version: 27264, sigs: 2060601, f-level: 90, builder: raynman)
main.cvd database is up-to-date (version: 62, sigs: 6647427, f-level: 90, builder: sigmgr)
bytecode.cvd database is up-to-date (version: 335, sigs: 86, f-level: 90, builder: raynman)
source venv/bin/activate;       \
	export PYTHONPATH=../src;       \
	cd test; pytest -s .
ERROR: Please define server type (local and/or TCP).
========================================================================= test session starts =========================================================================
platform darwin -- Python 3.10.11, pytest-8.2.0, pluggy-1.5.0
rootdir: /Users/dawood/git/ariel007/data-prep-lab/transforms/code/malware/test
plugins: anyio-4.3.0
collecting ... 09:24:28 INFO - Clamd process is not running. Start clamd process.
ERROR: Please define server type (local and/or TCP).
collected 1 item / 1 error 
```

Reply:
I will fix this